transport/system.go

```diff
@@ -99,19 +100,25 @@ func (t *System) buildOpenCmd() {
 
 // Open open a standard ssh connection.
 func (t *System) Open() error {
-	t.buildOpenCmd()
+	if t.OpenCmd == nil {
+		t.buildOpenCmd()
+	}
+
+	if t.ExecCmd == "" {
+		t.ExecCmd = "ssh"
+	}
 
 	logging.LogDebug(
 		t.FormatLogMessage(
 			"debug",
 			fmt.Sprintf(
 				"\"attempting to open transport connection with the following command: %s",
-				t.openCmd,
+				t.OpenCmd,
 			),
 		),
 	)
 
-	sshCommand := exec.Command("ssh", t.openCmd...)
+	sshCommand := exec.Command(t.ExecCmd, t.OpenCmd...)
```

Comment:
since it is now broader than just `ssh`, maybe we need to name the var something else, like

`command`